.gitignore

```diff
@@ -1,2 +1 @@
-dist
```

Comment:
sorry man, just working thought this.  Ill get it all cleaned up and let you know.  `npm link` does terrible things, so I have to push to github and `npm i` from my fork.